### DISCUSSION TITLE
Nightly Build doesn't connect properly

### DISCUSSION THREAD

**Discussion by marshr32** [winder/Universal-G-Code-Sender#1777] [Help] [answered]
Product Version: Universal Gcode Platform 20220114
Java: 13.0.1; OpenJDK Client VM 13.0.1+9
Runtime: OpenJDK Runtime Environment 13.0.1+9
System: Windows 10 version 10.0 running on x86; Cp1252; en_US (ugsplatform)

Trying the above build to see what's new. 
When I click connect it appears to connect, and brings up the alarm before homing as normal. However the Home button is grayed out, as are all others. I can't get to firmware settings either.
I've found though that by running the wizard and letting it connect there, everything works after that.

Everything also works fine with 2.0.9.

**Comment by breiler** [comment, marked as answer]
Thanks! The issue should now be resolved in the latest nightly build. 

When in an alarm state the controller is very limited what you are allowed to do (practically just "unlock", "home", "soft reset") so the other actions are therefore disabled. If you find any other problems in the nightly I would love to hear about them here!

**Reply by marshr32** [reply]
Thanks Joacim, it now connects just fine.
However, for me at least, the visualizer doesn't show any paths.

**Reply by breiler** [reply]
Thanks for the testing. I just tried the 2.0.10 build which should be equivalent to the nightly version that you are running. And it seems to work for me:
![image](https://user-images.githubusercontent.com/8962024/149901428-55e0b5e8-1307-4583-9fa0-10865ebf4323.png)

Do you mind sharing your log file from here: `C:/Users/[your username>]/.ugsplatform/2.X/var/log/messages.log`

**Comment by marshr32** [comment]
Ok I have no idea what the problem was before, but it seems to work today!
Sorry about that. I should have used the universal "did you restart your PC?" troubleshooting method first.

Interestingly the log file folder doesn't have anything created this year (I only downloaded and ran that nightly 2 days ago), so I'm unable to post that for you.

EDIT: After taking this screenshot I upgraded to 2.0.10, and that has made a folder today.

![image](https://user-images.githubusercontent.com/88805183/150012595-2dc2fdd3-b700-431b-b9e0-e99ad85ae4aa.png)

**Reply by breiler** [reply]
When running the nightly build the logs are placed under 2.0-SNAPSHOT. Let me know if the issue happens again.
